docs/includes/json-manifest-outlook-contextual-not-supported.md

```diff
@@ -1,2 +1,2 @@
 > [!NOTE]
-> Contextual Outlook add-ins aren't supported when the add-in uses a [Teams manifest for Office Add-ins (preview)](../develop/json-manifest-overview.md).
+> Contextual Outlook add-ins aren't supported when the add-in uses a [Unified Microsoft 365 manifest (preview)](../develop/json-manifest-overview.md).
```

Comment:
```suggestion
> Contextual Outlook add-ins aren't supported when the add-in uses a [unified Microsoft 365 manifest (preview)](../develop/json-manifest-overview.md).
```
Suggesting change for consistency. If you accept this, other links will need to be updated as well.

Reply:
That's also the title of the article, so I think I'll leave it capitalized in these links.